internal/k8s/kubernetes.go

```diff
@@ -290,3 +302,57 @@ func WaitForGitlab(dryRun bool, config *configs.Config) {
 	}
 	log.Printf("the output is: %s", output.String())
 }
+
+func RemoveSelfSignedCertArgoCD() error {
+	err := clearSecretField("argocd", "argo-secret", "/data/tls.crt")
+	if err != nil {
+		log.Printf("err removing tls.crt from argo-secret: %s", err)
+		return err
+	}
+	err = clearSecretField("argocd", "argo-secret", "/data/tls.key")
+	if err != nil {
+		log.Printf("err removing tls.crt from argo-secret: %s", err)
+		return err
+	}
+	return nil
+}
+
+// remove field from k8s secret using sdk
+func clearSecretField(namespace, name, field string) error {
+
+	secret := secret{
+		namespace: namespace,
+		name:      name,
+	}
+
+	payload := []PatchJson{{
+		Op:   "remove",
+		Path: field,
+	}}
+
+	clientset, err := GetClientSet(false)
+	if err != nil {
+		log.Panicf("Error creating k8s clientset : %s", err)
+		return err
```

Comment:
the error wont be returned since log.Panic will `print`, call `panic`, and stop the execution.

Reply:
Ok, I will change it to `printf` because if this process fails is not tragical and we can continue the process.